deepcell/layers/__init__.py

```diff
@@ -43,6 +43,7 @@
 from deepcell.layers.normalization import ImageNormalization3D
 from deepcell.layers.pooling import DilatedMaxPool2D
 from deepcell.layers.pooling import DilatedMaxPool3D
+from deepcell.layers.temporal import TempMerge, TempUnmerge
```

Comment:
I personally think these should be renamed to `TemporalMerge` and `TemporalUnmerge` for more clarity.

Reply:
Names have been updated